python/ql/src/Security/CWE-770/MissingRateLimiting.qhelp

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+HTTP request handlers should not perform expensive operations, such as accessing
+the file system, executing an operating system command, or interacting with a database,
+without limiting the rate at which requests are accepted. Otherwise, the application
+becomes vulnerable to denial-of-service attacks where an attacker can cause the
+application to crash, or become unresponsive, by issuing a large number of requests at
+the same time.
+</p>
+</overview>
+
+<recommendation>
+<p>
+A rate-limiting middleware should be used to prevent such attacks.
+</p>
+</recommendation>
+
+<example>
+<p>
+Assuming that <code>expensive_handler</code> is designated as an expensive handler,
+a declaration of this sort will be flagged:
+</p>
+
+<source>
```

Comment:
Thanks for adding an example. For details on what elements are allowed in a qhelp file, see https://help.semmle.com/QL/learn-ql/writing-queries/query-help.html.

It looks as if `<source>` should be `<sample language=python>`.